updater/flasher.go

```diff
@@ -22,16 +22,36 @@ import (
 	"strings"
 
 	"github.com/arduino/go-paths-helper"
+	"github.com/shirou/gopsutil/v4/disk"
 
 	"github.com/arduino/arduino-flasher-cli/feedback"
 	"github.com/arduino/arduino-flasher-cli/i18n"
 	"github.com/arduino/arduino-flasher-cli/updater/artifacts"
 )
 
-func Flash(ctx context.Context, imagePath *paths.Path, version string, forceYes bool) error {
+const GiB = uint64(1024 * 1024 * 1024)
+const DownloadDiskSpace = uint64(12)
+const ExtractDiskSpace = uint64(10)
```

Comment:
I would keep two values. One for only the estimated archive size and another for the extract size
```suggestion
const EstimatedDownloadSize = uint64(2) * GiB
const EstimateExtractionSize = uint64(10) * GiB
```